pages/public_cloud/data_analytics/analytics/analytics_kubernetes_operator/guide.en-gb.md

```diff
@@ -0,0 +1,147 @@
+- A [Public Cloud project](/pages/public_cloud/compute/create_a_public_cloud_project) in your OVHcloud account
+
+### Getting your OVHcloud API tokens information
+
+In order to generate your OVHcloud API, please follow our [First steps with the OVHcloud APIs](/pages/manage_and_operate/api/first-steps) tutorial.
+
+Specifically, you have to generate these credentials via the [OVHcloud token generation page](https://api.ovh.com/createToken?GET=/cloud/project/*/database/*&POST=/cloud/project/*/database/*&PUT=/cloud/project/*/database/*&DELETE=/cloud/project/*/database/*) with the following rights:
+
+- `GET /cloud/project/*/database/*`
+- `POST /cloud/project/*/database/*`
+- `PUT /cloud/project/*/database/*`
+- `DELETE /cloud/project/*/database/*`
+
+## Instructions
+
+The Kubernetes database operator is stored as an Helm chart in [Docker Hub](https://hub.docker.com/r/ovhcom/public-cloud-databases-operator/tags), an OCI registry.
+
+### Helm Values
+
+The first step is to create a `values.yaml`. 
+
+Please replace the `XXXX` with the credentials you previously retrieved.
+
+Note that the region value is either `ovh-eu`, `ovh-ca` or `ovh-us`.
+
+```yaml
+ovhCredentials:
+  applicationKey: XXXX
+  applicationSecret: XXXX
+  consumerKey: XXXX
+  region: ovh-eu # ovh-eu, ovh-ca or ovh-us
+
+namespace: ovhcloud # Your Kubernetes namespace
+```
+
+### Installation
+
+Use the kubernetes package manager [helm](https://helm.sh) and the values file you created to install the operator.
+
+```bash
+helm install -f values.yaml public-cloud-databases-operator oci://registry-1.docker.io/ovhcom/public-cloud-databases-operator --version 0.1.1
+```
+
+This command will create the operator, CRDs and secrets objects.
+
+You can find the latest version of the helm chart on [DockerHub](https://hub.docker.com/r/ovhcom/public-cloud-databases-operator/tags).
+
+You can verify the operator is correctly installed by checking the Pods in the new Kubernetes namespace:
+
+```bash
+kubectl get deploy -n ovhcloud
+NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
+public-cloud-databases-operator   1/1     1            1           60m
+```
+
+And that the secret with you OVHcloud credentials is properly created:
+
+```bash
+kubectl get secret ovh-credentials -n ovhcloud
+NAME              TYPE     DATA   AGE
+ovh-credentials   Opaque   4      60m
+```
+
+### Creating custom resources
+
+Create a custom resource object:
+
+```yaml
+apiVersion: cloud.ovh.net/v1alpha1
+kind: Database
+metadata:
+  name: myanalyticsservice
+  namespace: ovhcloud
+spec:
+  projectId: XXXX
+  serviceId: XXX
+```
+
+Or with a label selector to cherry pick you nodes based on label:
+
+```yaml
+apiVersion: cloud.ovh.net/v1alpha1
+kind: Database
+metadata:
+  name: myanalyticsservice
+  namespace: ovhcloud
+spec:
+  projectId: XXXX
```

Comment:
projectID is what and seviceID is what?
It will be useful for the viewer to know if projectID is related to the database or to the clusterID, is not easy to understand what information he/she must fill :)